src/unstract/sdk/file_storage/constants.py

```diff
@@ -0,0 +1,9 @@
+class Common:
+    FULL = -1
+    DEFAULT_ENCODING = "utf-8"
```

Comment:
```suggestion
    READ_ENTIRE_FILE = -1
    UTF_8 = "utf-8"
```
NIT: `Common` doesn't feel like a good name to define these constants. I suggest moving this within the `FileStorage` class where its actually associated / meant to be used. I've also suggested some alternate names to convey the use better

Reply:
`Full` is might be  better to change to `READ_ENTIRE_FILE`. but `UTF_8` not seems a  better replaceble for  DEFAULT_ENCODING. If we need UTF_8 we should create a new class for constants like ENCODINGS .

Reply:
Done